androidHyperskillApp/src/main/java/org/hyperskill/app/android/main/view/ui/activity/BackToForegroundObserver.kt

```diff
@@ -0,0 +1,26 @@
+package org.hyperskill.app.android.main.view.ui.activity
+
+import androidx.lifecycle.DefaultLifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+
+class BackToForegroundObserver(
```

Comment:
@XanderZhu `HomeFragment` has an [onForegroundObserver](https://github.com/hyperskill/mobile-app/blob/develop/androidHyperskillApp/src/main/java/org/hyperskill/app/android/home/view/ui/fragment/HomeFragment.kt#L77-L83), maybe also migrate it to the `BackToForegroundObserver` ?